app-backend/common/src/main/scala/utils/CogUtils.scala

```diff
@@ -172,4 +172,43 @@ object CogUtils {
       hists
     }
   }
+
+  def geoTiffDoubleHistogram(tiff: GeoTiff[MultibandTile], buckets: Int = 80, size: Int = 128): Array[Histogram[Double]] = {
```

Comment:
Not super happy with this solution (was in a little bit of a rush) so if there's a better idea than just duplicating the streaming histogram method I'm open to it. There were a lot of consequences of trying to switch everything to streaming histograms so this was the quickest, least invasive method I could come up with.

Reply:
Probably the place for effort for improving tile-server strategy is over in geotrellis/geotrellis-server, so :man_shrugging: